lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -35,6 +35,9 @@ class EstimatedInputLatency extends Audit {
       name: 'estimated-input-latency',
       description: 'Estimated Input Latency',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
+      helpText: 'Primarily a measure of the main thread\'s activity. ' +
+      'There is a 90% probabililty a user would encounter of this amount or less. ' +
```

Comment:
"There is a 90% probabililty a user would encounter *input latency* of this amount or less"?